src/index.html

```diff
@@ -4,9 +4,409 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,400;1,500;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          class="header__img"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и Телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#"
+            data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="cards">
+        <div class="card" data-qa="card">
+          <div class="card__photo">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac"
+              class="card__photo-img"
+            >
+          </div>
+          <div class="card__content">
+            <div class="card__description">
+              <h2 class="card__description-text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+            </div>
+            <div class="card__product-code">
+              <p class="card__product-code-text">Код товара: 195434</p>
+            </div>
+            <div class="card__review">
+              <div class="stars">
```

Comment:
add modifier stars--4 here, and write styles for all the possible modifiers, so that changing the modifier in html to stars--3 would change number of active stars accordingly

Reply:
done